integrations/tensorflow/e2e/keras/train/BUILD

```diff
@@ -28,10 +29,27 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
+[
+    iree_py_binary(
+        name = src.replace(".py", "_manual"),
+        srcs = [src],
+        main = src,
+        python_version = "PY3",
+        deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+            "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+        ],
+    )
+    for src in glob(["*_test.py"])
+]
+
 iree_e2e_cartesian_product_test_suite(
-    name = "train_tests",
-    srcs = ["model_train_test.py"],
+    name = "classification_training_tests",
+    srcs = ["classification_training_test.py"],
     failing_configurations = [
+        {
+            # TFLite doesn't support training.
+            "target_backends": "tflite",
+        },
```

Comment:
Wouldn't it be easier to just remove tflite from target_backends entirely? Are we likely to want to test in the future whether tflite supports training with the `_failing` targets? Seems like not to me. We shouldn't be debugging tflite in that way.

Reply:
Thankfully it's not our job to get `TFLite` working in any of our tests :P. But I think we should keep this as is because:

1. This allows us to create a coverage table for these tests in the future.
2. TFLite _may_ eventually have a backend which supports training, and if that happens we would want to reflect that in our coverage tables as well.

Reply:
Ok I buy (1). For (2) we could just add it at that time, so I don't find that very compelling :-D